modules/sso-rbac/README.md

```diff
@@ -0,0 +1,106 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
please have some minimal description what this module is for,
the example file in ./examples folder seems not having reference to this module, if you do not need this file please remove it